grpc-sys/bindings/x86_64-unknown-linux-gnu-bindings.rs

```diff
@@ -706,6 +672,7 @@ fn bindgen_test_layout_grpc_slice() {
 #[doc = " Represents an expandable array of slices, to be interpreted as a"]
 #[doc = "single item."]
 #[repr(C)]
+#[derive(Copy, Clone)]
 pub struct grpc_slice_buffer {
```

Comment:
Is it ok to copy the slice? How do we manage the ownership of allocated memory?

Reply:
Yes, it's a bit-wise copy of the struct value, so users have to deal with ownership carefully. I think it's OK for the C type to be copy and clone. For example, `libc::iovec` is also `Copy` and `Clone`, but calling clone will not actually clone the inner data. Make it `Copy` is handy and necessary for fighting with FFIs.